setup.cfg

```diff
@@ -32,3 +32,6 @@ test =
 [options.entry_points]
 pytest11 =
     pytest_astropy_header = pytest_astropy_header.display
+
+[flake8]
+max-line-length = 125
```

Comment:
how come this number? 

(it's OK, I just wonder why it's not 80 or 100 or 120, 125 is honestly just as good, just a bit unusual)

Reply:
I can try 120 but pretty sure my Emacs would still complain if it is 100.

Reply:
Answer: It was arbitrarily chosen to silence `flake8`.

Reply:
125 it is then. I really just wondered 😄

Reply:
If you like power of 2, I can increase to 128. Or even 129 if you want 128 *without* the null terminator. :smirk: